yarn.lock

```diff
@@ -8919,6 +8919,7 @@ __metadata:
   dependencies:
     "@suite-common/suite-config": "workspace:*"
     "@suite-common/suite-types": "workspace:*"
+    "@suite-common/wallet-config": "workspace:*"
```

Comment:
should be part of `feat(suite): display account type badges` commit

Reply:
probably part of `feat(suite): filter accounts based on debug type` as you added the lib because of this, so all the logic related to adding lib because of filtering debug type accounts should be part of `feat(suite): filter accounts based on debug type`

Reply:
probably part of `feat(suite): filter accounts based on debug type` as you added the lib because of this, so all the logic related to adding lib because of filtering debug type accounts should be part of `feat(suite): filter accounts based on debug type`